robotbase/src/main/java/com/team2357/frc2022/subsystems/TurretSubsystem.java

```diff
@@ -91,22 +92,26 @@ public void resetHeading() {
         m_turretMotor.getEncoder().setPosition(0);
     }
 
-    public double getTurretRotations() {
-        return m_turretMotor.getEncoder().getPosition();
+    public double getTurretDegrees() {
+        return m_turretMotor.getEncoder().getPosition() / m_config.m_rotationsPerDegree;
     }
 
-    public void setTurretPosition(double position) {
-        m_setPoint = position;
-        m_pidController.setReference(m_setPoint, CANSparkMax.ControlType.kSmartMotion);
+    /**
+     * 
+     * @param targetDegrees Desired position of the turret in degrees
+     */
+    public void setTurretPosition(double targetDegrees) {
+        m_targetDegrees = targetDegrees;
```

Comment:
This function should check if we are zeroed or not. If we don't have a zero, we can't go closed loop. I'd suggest returning true if this function is successful, false if it fails (and make sure to log).

Reply:
Changed it so that this method requires closed loop.